Colorspin.xcodeproj/project.pbxproj

```diff
@@ -1051,13 +1051,13 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = PK94RJWMB9;
```

Comment:
Don't commit your changes to this file, as I can't use your Apple ID.

We have to setup a developer team profile soon...

Reply:
Also, don't commit .xcuserdata files .. let's just keep using the shared data profile